lib/main.js

```diff
@@ -284,14 +284,24 @@ const Hydrogen = {
     this.existingKernelPicker.toggle();
   },
 
-  handleKernelCommand(
-    { command, payload }: { command: string, payload: ?Kernelspec },
-    { kernel, markers }: { kernel: ?Kernel, markers: ?MarkerStore }
-  ) {
+  handleKernelCommand({
+    command,
+    payload
+  }: {
+    command: string,
+    payload: ?Kernelspec
+  }) {
     log("handleKernelCommand:", arguments);
 
+    const { kernel, grammar, markers } = store;
+
+    if (!grammar) {
+      atom.notifications.addError("Undefined grammar");
+      return;
+    }
+
     if (!kernel) {
-      const message = "No running kernel for grammar or editor found";
```

Comment:
@wadethestealth 
I just made this change since I reverted this file from the old master state, but why did you make this change ?
I just found this here.

Reply:
In my opinion there is no reason to list out the grammar, and it required an extra argument